core/templates/search/search_pages_results.html

```diff
@@ -221,6 +222,7 @@ <h4 class="term">{{ paginator.count }} results
 
 {% block search_results_box %}
 <div class="content search_results">
+  <h3>Newspaper Pages</h3>
```

Comment:
If I remember right, we were not calling the results "Newspaper" ... I think "Pages" would get the point across and not introduce something that could be incorrect in other contexts.